explainer/index.bs

```diff
@@ -1209,7 +1209,7 @@ A typical bitflag definition looks like this:
 <xmp highlight=idl>
 typedef [EnforceRange] unsigned long GPUColorWriteFlags;
 [Exposed=Window]
```

Comment:
can we remove all the Exposed= attributes now? or do they still apply to namespaces?

Reply:
They still apply (and are required)